project/src/components/add-review-form/add-review-form.tsx

```diff
@@ -0,0 +1,45 @@
+import {useState} from 'react';
+import StarsRating from '../stars-raiting/stars-raiting';
+
+function AddReviewForm(): JSX.Element {
+  const [rating, setRating] = useState(0);
+  const [review, setReview] = useState('');
+
+  const handleInputChange = (item: number): void => setRating(item);
+
+  const isDisabled = (review.length < 50 || review.length > 400) || rating === 0;
+
+  return (
+    <form action="#" className="add-review__form">
+      <div className="rating">
+        <div className="rating__stars">
+          {Array.from({length:10}, (k, i) => (
```

Comment:
```suggestion
          {Array.from({length: 10}, (k, i) => (
```

10 - магическое число, вынеси в константу